### PR TITLE
Feature/32 page add course

### PR DESCRIPTION
**Pull request by FHToE**

### CODE REVIEW

File: frontend/src/containers/Routing/index.tsx [hunk 1]
```diff
@@ -31,6 +33,10 @@ const Routing: React.FunctionComponent<IRoutingProps> = ({ isLoading }) => (
       <PublicRoute exact path="/" component={LandingPage} />
       <PublicRoute exact path="/login" component={LoginPage} />
       <PublicRoute exact path="/main" component={MainStudentPage} />
+      <PublicRoute exact path="/public" component={Data} />
+      <PublicRoute exact path="/add_path" component={AddPathPage} />
+      <PublicRoute exact path="/landing" component={LandingPage} />
+      <PublicRoute exact path="/add_course" component={AddCourse} />
       <PublicRoute exact path="/oauth2/redirect" component={handler} />
```
Comment (MaximGoncharukSW) [inline frontend/src/containers/Routing/index.tsx]: Rename it to `<PublicRoute exact path="/oauth2/redirect" component={handler} />` or update with the latest dev.

File: frontend/src/components/TagSelector/react_tags.sass [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+ *         <li class="is-active">
+ *           <mark />
+ *         </li>
+ *         <li class="is-disabled">
+ *           <mark />
+ *         </li>
+ *       </ul>
+ *     </div>
+ *   </div>
+ */
```
Comment (MaximGoncharukSW) [inline frontend/src/components/TagSelector/react_tags.sass]: Why do you need this set of comments?

File: frontend/src/screens/AddCourse/services/course.service.ts [hunk 3]
```diff
@@ -1,10 +1,27 @@
...
   });
+  const result = response.json();
+  return result;
+};
+
+/* export const getLecturesByAuthor = async (id: string) => new Promise(resolve => {
+  setTimeout(() => resolve({
+    lectures
+  }), 500);
+}); */
```
Comment (MaximGoncharukSW) [inline frontend/src/screens/AddCourse/services/course.service.ts:18]: Do you need this comment?

Comment (FHToE) [reply frontend/src/screens/AddCourse/services/course.service.ts:18]: yes, this method exists to fetch data from mock

File: frontend/src/components/TagSelector/react_tags.sass (hunk 2 above)
Comment (FHToE) [reply frontend/src/components/TagSelector/react_tags.sass]: unfortunaly, this code wasn't written by me, i pulled it from none-dev branch to reusing components and havent resolve yet its purpose

Comment (MaximGoncharukSW) [reply frontend/src/components/TagSelector/react_tags.sass]: Just get rid of it.

Comment (FHToE) [reply frontend/src/components/TagSelector/react_tags.sass]: Done, master!)

**Review by MaximGoncharukSW — APPROVED** [review]

**Pull request merged**